src/car_rental_topic.py

```diff
@@ -0,0 +1,113 @@
+def total_kms(odometer_start: float, odometer_finish: float) -> float:
+    """
+    This function calculates the total number of kilometers driven based
+    on starting and ending odometer readings.
+
+    @rtype: float
+    @param odometer_start: The number of kms the car had before renting
+    @param odometer_finish: The number of kms the car had after rending
+    @return: The total kms driven
+    """
+
+    return odometer_finish - odometer_start
+
+
+assert 0 == total_kms(0, 0)
+assert 100 == total_kms(0, 100)
+assert -100 == total_kms(100, 0)
+assert 100.5 == total_kms(100.5, 201)
```

Comment:
Why include assertions and `unittest` tests?

Reply:
- Assertions are the ones they will see at this stage. 
- Unit tests are the "real" tests that also serve as reference for the keeners that want to see the REAL way
- Later in the course, the unittests will be shown as it does work on COLAB.

Reply:
Got it. Makes sense